pages/your_tasks/software_documentation.md

```diff
@@ -124,3 +127,5 @@ for some automated tasks.
 3. [Perez-Riverol Y, Gatto L, Wang R, Sachsenberg T, Uszkoreit J, Leprevost FdV, et al. Ten Simple Rules for Taking Advantage of Git and GitHub](https://doi.org/10.1371/journal.pcbi.1004947)
 4. [How to Write Software Documentation in 7 Simple Steps](https://technicalwriterhq.com/documentation/software-documentation/how-to-write-software-documentation/)
 
+
+[open-ai]: https://swagger.io/specification/

```

Comment:
Related to my previous comment, this might rather be `
[open-api]: https://swagger.io/specification/`

Reply:
Thank you! Got it now!